staging/src/k8s.io/apiserver/pkg/server/filters/wrap.go

```diff
@@ -51,16 +50,15 @@ func WithPanicRecovery(handler http.Handler, resolver request.RequestInfoResolve
 			// This call can have different handlers, but the default chain rate limits. Call it after the metrics are updated
 			// in case the rate limit delays it.  If you outrun the rate for this one timed out requests, something has gone
 			// seriously wrong with your server, but generally having a logging signal for timeouts is useful.
-			runtime.HandleError(fmt.Errorf("timeout or abort while handling: %v %q", req.Method, req.URL.Path))
+			runtime.HandleError(fmt.Errorf("timeout or abort while handling: method=%v URI=%q audit-ID=%q", req.Method, req.RequestURI, request.GetAuditID(req)))
 			return
 		}
 		http.Error(w, "This request caused apiserver to panic. Look in the logs for details.", http.StatusInternalServerError)
```

Comment:
should we change the msg to contain the correlation ID as well? Users won't see it by default only with increased log level.

Reply:
well, the audit ID is set in the response header. Also, for when net/http catches the panic it will close the connection or reset the stream for http2. So the user doesn't see the http response, I think.